mediapipe/tasks/ios/vision/core/utils/sources/MPPImage+Utils.mm

```diff
@@ -178,24 +178,48 @@ @implementation MPPPixelDataUtils : NSObject
                                       static_cast<uint8_t *>(destBuffer.data));
 }
 
-+ (UInt8 *)pixelDataFromImageFrame:(ImageFrame &)imageFrame
-                        shouldCopy:(BOOL)shouldCopy
-                             error:(NSError **)error {
++ (UInt8 *)rgbaPixelDataFromImageFrame:(ImageFrame &)imageFrame
+                            shouldCopy:(BOOL)shouldCopy
+                                 error:(NSError **)error {
   vImage_Buffer sourceBuffer = CreateVImageBufferFromImageFrame(imageFrame);
 
   // Pre-multiply the raw pixels from a `mediapipe::Image` before creating a `CGImage` to ensure
   // that pixels are displayed correctly irrespective of their alpha values.
-  vImage_Error premultiplyError;
   vImage_Buffer destinationBuffer;
+  vImage_Error vImageOperationError;
 
   switch (imageFrame.Format()) {
     case ImageFormat::SRGBA: {
       destinationBuffer =
           shouldCopy ? CreateEmptyVImageBufferFromImageFrame(imageFrame, true) : sourceBuffer;
-      premultiplyError =
+      vImageOperationError =
           vImagePremultiplyData_RGBA8888(&sourceBuffer, &destinationBuffer, kvImageNoFlags);
       break;
     }
+    case ImageFormat::SRGB: {
+      // Some tasks like the Face Stylizer output RGB images inspite of the input being restricted
```

Comment:
@schmidt-sebastian For the reason mentioned here, it's not possible to have a method that returns un-copied stylised image buffer in iOS.

I have removed the method from the interface.